docs/modules/storage/pages/configuring-persistence.adoc

```diff
@@ -1024,6 +1024,14 @@ include::ROOT:example$/storage/SampleEncryptionAtRestConfiguration.java[tag=vaul
 
 Use the following options to configure persistence for xref:data-structures:map.adoc[map] and xref:jcache:overview.adoc[JCache] data structures.
 
+[NOTE]
+====
+To avoid exception errors, make sure that your map names do not:
```

Comment:
```suggestion
You cannot persist maps if their names contain either of the following. Otherwise, Hazelcast throws an exception.
```

The important point is that you cannot configure persistence for certain maps, depending on their name. The fact that such maps will cause exceptions is secondary information.